101-function-app-create-dedicated/azuredeploy.json

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://schemas.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
```

Comment:
Use the later schema "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",